client/src/lib/GlobalStyles.js

```diff
@@ -7,10 +7,11 @@ export const BaseCSSReset = css`
   html {
     font-size: 16px;
     line-height: 1.25;
+    min-height: 100vh;
   }
   body {
     min-height: 100vh;
-    background-color: #F2E7FF;
+    background-color: #D6C7C0;
```

Comment:
This is one of our color variables, which are located in `GlobalStyles` starting on line 126. You can replace `#D6C7C0` with `var(--peachShade1)`